reference/ToDo/src/ToDo/Views/SearchPage.xaml

```diff
@@ -1,133 +1,131 @@
+                      Background="{ThemeResource SurfaceBrush}">
+        <Button uen:Navigation.Request="-"
+                utu:AutoLayout.CounterAlignment="Start"
+                Style="{StaticResource IconButtonStyle}"
+                x:Uid="Body_IconButton">
+          <PathIcon Data="{StaticResource Icon_Arrow_Back}" />
+        </Button>
+        <TextBox x:Uid="SearchPage_InputTextBox"
+                 Text="{Binding Term, Mode=TwoWay, UpdateSourceTrigger=PropertyChanged}"
+                 PlaceholderText="Search"
+                 utu:AutoLayout.CounterAlignment="Start"
+                 utu:AutoLayout.PrimaryAlignment="Stretch"
+                 Background="{ThemeResource SurfaceBrush}"
+                 Style="{StaticResource FilledTextBoxStyle}" />
+      </utu:AutoLayout>
+      <uer:FeedView Source="{Binding Results}"
+                    utu:AutoLayout.CounterAlignment="Start"
+                    utu:AutoLayout.PrimaryAlignment="Stretch">
+        <DataTemplate>
+          <ListView Margin="20"
+                    ItemsSource="{Binding Data}"
+                    uen:Navigation.Request="Task"
+                    Style="{StaticResource ListViewStyle}">
+            <ListView.ItemsPanel>
+              <ItemsPanelTemplate>
+                <ItemsStackPanel Orientation="Vertical" />
+              </ItemsPanelTemplate>
+            </ListView.ItemsPanel>
+            <ListView.ItemTemplate>
+              <DataTemplate>
+                <SwipeControl Margin="5">
+                  <SwipeControl.RightItems>
+                    <SwipeItems Mode="Reveal">
+                      <SwipeItem Foreground="{ThemeResource SurfaceBrush}"
+                                 Background="{ThemeResource ErrorBrush}">
+                        <SwipeItem.IconSource>
+                          <PathIconSource Data="{StaticResource IconSource_Delete_Outline}" />
+                        </SwipeItem.IconSource>
+                      </SwipeItem>
+                    </SwipeItems>
+                  </SwipeControl.RightItems>
+                  <SwipeControl.Content>
+                    <utu:AutoLayout PrimaryAxisAlignment="Center"
+                                    Background="{ThemeResource OnPrimaryBrush}">
+                      <utu:AutoLayout Spacing="5"
+                                      Orientation="Horizontal">
+                        <utu:AutoLayout Padding="10,0,0,0"
+                                        utu:AutoLayout.CounterAlignment="Center">
+                          <utu:AutoLayout Spacing="16"
+                                          Orientation="Horizontal"
+                                          utu:AutoLayout.CounterAlignment="Start"
+                                          utu:AutoLayout.CounterLength="40">
+                            <CheckBox IsChecked="{Binding IsCompleted}"
+                                      Command="{controls:AncestralBinding Path=ToggleIsCompleted, AncestorType=uer:FeedView}"
```

Comment:
This should start using the AncestorBinding from Toolkit, as well as the ItemsControlBindinfg

Currently it's using a local version, which I'm assuming was there before it moves to Toolkit

Reply:
yes you are right, I will do the update

Reply:
Switched to use `AncestorBinding` from Toolkit. Local previous version was removed for `AncestralBindingExtension` and `ItemsControlBindingExtension`.